contracts/vaults/Withdraw.sol

```diff
@@ -46,7 +46,7 @@ contract Withdraw is Base {
         updatePool(_pid);
 
         uint256 pending = ((user.shares * pool.rewardPerShare) / 1e12) - user.rewardDebt;
```

Comment:
we should call _claimReward here

Reply:
Yes but the claim reward has some checks which are unnecessary so not sure because this duplication could save some gas